lib/utils.pm

```diff
@@ -2073,4 +2074,32 @@ sub cleanup_disk_space {
     assert_script_run('snapper -croot set-config NUMBER_LIMIT=5-10');
 }
 
+
+=head2 package_upgrade_check
+    package_upgrade_check();
+
+This function is used for checking if the package
+is upgraded to the required version
+
+Sample config of parameter of the function below:
+my $pkg_list = {ibmtss => '1.6.0'};
+here, 'libmtss' is the package name, and '1.6.0'
+is target version which needs to be upgrated to
+
+=cut
+
+sub package_upgrade_check {
+    my ($pkg_list, $fail_flag) = @_;
```

Comment:
No default value for $fail_flag.

Reply:
If we don't define a value, it should be null, correct? then we will not need to check it.

Please share me some light if any suggest code changes?

Reply:
> If we don't define a value, it should be null, correct? then we will not need to check it.
> 
> Please share me some light if any suggest code changes?

You are correct, the code is OK too.

Reply:
Thanks Lili, then let me keep the current logic